articles/how-to-guides/create-a-designer-extension/create-a-designer-extension.md

```diff
@@ -0,0 +1,71 @@
+---
+uid: how-to-guides.create-a-designer-extension
+note: This one will only cover the new element creation. We will need to write one that will form part of 'creating a new designer' that will allow you to define your visuals too.
+---
+# Create a Designer Extension
+
+If you have worked in Intent Architect's available set of designers such as the Services or the Domain designer, chances are you might have asked yourself the question whether you could change them to some extent.
+In this how-to you will learn how to add your own Element to a Domain designer (or any other designer) and be able to reference it in the designer as if it is an actual type and be able to generate content from that new element.
```

Comment:
Add a new line above this one to make this a new paragraph, or alternatively remove the line break which has no effect once rendered.